18F/identity-idp
# LG-12636: Show recommended PIV option for .gov/.mil email

## 🎫 Ticket

[LG-12636](https://cm-jira.usa.gov/browse/LG-12636)

## 🛠 Summary of changes

Recommends PIV as an MFA option in account creation for users with a ".gov" or ".mil" email:

- Sorts the option to appear first in the selection list
- Displays a "Recommended" tag

## 📜 Testing Plan

Repeat the following steps with and without a ".gov" or ".mil" email:

1. Go to http://localhost:3000
2. Click "Create an account"
3. Complete account creation up until MFA selection
4. Observe:
   - If you entered a ".gov" or ".mil" email for your account, PIV appears first and has a "Recommended" tag
   - Otherwise, PIV appears last and does not have a "Recommended" tag
   - All other options are unaffected

## 👀 Screenshots

Before|After
---|---
![localhost_3000_authentication_methods_setup(Computer Standard) (1)](https://github.com/18F/identity-idp/assets/1779930/9095a7f6-7e49-4d0d-9948-02f335bf8492)|![localhost_3000_authentication_methods_setup(Computer Standard)](https://github.com/18F/identity-idp/assets/1779930/c8c4e647-57b7-4062-b7cd-c96802ce5c68)
